python/lsst/ctrl/execute/allocator.py

```diff
@@ -303,6 +305,13 @@ def getNodes(self):
         """
         return self.getParameter("NODE_COUNT")
 
+    def getMemoryPerCore(self):
+        """Accessor for MemoryPerCore
+        @return the value of MemoryPerCore
+        """
+        MemoryPerCore = 4096
```

Comment:
How can this be overridden?  Or if not, why not?

Reply:
This was "starting down the road" to supporting users specifying the Memory Per Core. 
The whole solution hasn't been worked through yet, but I wanted to get some kind 
of explicit --mem into the sbatch suhmissions because, for example, the 
Slurm scheduler at CC-IN2P3 requires a  --mem setting to submit, and so this 
allows initial testing there.